### PR TITLE
fix categorized filters

### PR DESCRIPTION
**Pull request by dauglyon**

### CODE REVIEW

**Comment by codytodonnell** [comment]
This looks good, filters are restored. 

I'm noticing a new issue now where the state of the filter components (e.g. the sliders) doesn't update when switching between contexts when the filter panel remains open. If you close and reopen the panel then the inputs update as expected.

**Comment by dauglyon** [comment]
Hmm, will have to add a context dependency to whatever effect is controlling the state, I'll take a look

File: src/features/collections/CollectionDetail.tsx
Comment (codytodonnell) [inline src/features/collections/CollectionDetail.tsx:1]: Ah interesting way to solve this issue

**Review by codytodonnell — APPROVED** [review]
Looks good!

**Pull request merged**